packages/button-theme/stories/Button.stories.tsx

```diff
@@ -15,55 +15,50 @@ export default {
         options: ['small', 'medium', 'large'],
       },
     },
+    variant: {
+      description: 'The style variant to use.',
+      control: {
+        type: 'select',
+        options: ['primary', 'secondary', 'warning', 'dark'],
+      },
+    },
   },
 } as Meta;
 
-const Template: Story = args => (
+const JSTemplate: Story = args => (
+  <>
+    <ButtonTypography />
+    <Button {...args}>Button</Button>
+  </>
+);
+
+const HTMLTemplate: Story = args => (
   <>
     <ButtonTypography />
-    <h3>HTML Only</h3>
     <HtmlOnlyWrapper>
       <Button {...args}>Button</Button>
     </HtmlOnlyWrapper>
+  </>
+);
 
-    <Divider />
-
-    <h3>HTML + CSS</h3>
+const CSSTemplate: Story = args => (
+  <>
+    <ButtonTypography />
     <HtmlWithCssWrapper>
       <Button {...args}>Button</Button>
     </HtmlWithCssWrapper>
-
-    <Divider />
-
-    <h3>HTML + CSS + JS</h3>
-    <Button {...args}>Button</Button>
   </>
 );
-
-export const Primary = Template.bind({});
```

Comment:
In files that had stories for each variant, I added the variants to the default export so you can view them in the js/css/html stories